### PR TITLE
Release 2022-10-11

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Release of 11th of October of 2022

# Changes

* Update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Proposal XXXXX
+* Move to SvelteKit.
```
Comment (peterpeterparker) [inline CHANGELOG.md:4]: Can you add something like "(without routing changes)"?

> * Move to SvelteKit (without routing changes)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+* New transaction modal to top up neurons.
+* Upgrade agent-js libraries.
```
Comment (peterpeterparker) [inline CHANGELOG.md:6]: `v0.14.0`

File: CHANGELOG.md (hunk 1 above)
Comment (lmuntaner) [reply CHANGELOG.md:4]: yes

File: CHANGELOG.md (hunk 2 above)
Comment (lmuntaner) [reply CHANGELOG.md:6]: sure

**Review by peterpeterparker — APPROVED** [review]
Thx

**Comment by lmuntaner** [comment]
Reproducibility was lost in this one.

**Pull request closed without merging**